nvidia-driver-installer/cos/daemonset-preloaded-latest.yaml

```diff
@@ -116,8 +116,8 @@ spec:
             echo "GPU driver is already installed, the installed version may or may not be the driver version being tried to install, skipping installation"
             exit 0
           else
-            echo "No GPU driver module detected, installting now"
-            /cos-gpu-installer install --version=latest
+            echo "No GPU driver module detected, installing now"
+            /cos-gpu-installer install --version=latest && exit 0 || exit 1
```

Comment:
should we continue running if command succeeds? and better printing an error message as well before exit 1.

            /cos-gpu-installer install --version=latest || exit 1